dev/ssl.go

```diff
@@ -98,7 +83,36 @@ func SetupOurCert() error {
 
 	keyOut.Close()
 
-	return TrustCert(certPath)
+	return nil
```

Comment:
this diff is a bit hard to read, but I just refactored out most of the `SetupOurCert` function into `GeneratePumaDevCertificateAuthority`